### DISCUSSION TITLE
Need to mention Docker BuildKit is needed to build Dockerfile

### DISCUSSION THREAD

**Discussion by Ming-D-BigData** [apache/airflow#24502] [General]
I checked out from main branch. Ran "docker build ." on Dockerfile in Ubuntu.  It gave me an error about syntax problem in Dockerfile, regarding the use of heredoc with COPY commands.
It costed me some time to figure out I needed to do "$DOCKER_BUILDKIT=1 docker build .". Or instead I needed to set the Docker BuildKit option permanently.

This does not seem to be mentioned on the front page of GitHub project. Or did I just miss it?

**Comment by potiuk** [comment]
It's mentioned in the changelog https://airflow.apache.org/docs/docker-stack/changelog.html

But If you feel that it needs to be elsewhere - feel absolutely free to provide a PR so that others do not stumble upon the same problem you did. And you are absolutely the best person to know where and how to mention it.  Because you have not found it before.

So please - by all means make a PR to add such mentioning @Ming-D-BigData . Airflow is created by more than 2000 contributors like you and a lot of valuable doc contribution came from people like you who were frustrated because they could not find something easily.

And it's supeer easy. All our Doc pages have "Suggest a change on this page" button at the bottom-right. Just click it, and "pull request"  will open, where you will be able to submit your change. As easy as opening issue or discussion here.

Looking forward to your PR. You can ping me when you open it.

**Reply by Ming-D-BigData** [reply]
@potiuk Will do later this week when I have some time for it. Strange thing is build passed that point(didn't come across same error) when I was trying to build same Dockerfile on Windows platform. 

thanks,

**Reply by potiuk** [reply]
Docker Desktop on Windows has buildkit enabled by default.
